vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.java

```diff
@@ -1715,47 +1688,19 @@ private void refreshEmailsList() {
             // add new emails list
             mDisplayedEmails = newEmailsList;
 
-            int index = 0;
-            final Preference addEmailBtn = mUserSettingsCategory.findPreference(ADD_EMAIL_PREFERENCE_KEY);
-
-            // reported by GA
-            if (null == addEmailBtn) {
-                return;
+            if(null != newEmailsList && mDisplayedEmails.size() > 1) {
```

Comment:
This is a bit weird to make this 2 tests on two different variables. I know they host the same value but if someone introduce some code just before the `if` in the future, these 2 variables can become different and we will win a bug.

Need also a space between `if` and `(`.